docs/user-guide.md

```diff
@@ -207,13 +242,21 @@ Approve or reject a model by clicking the associated button.
 
 ### When should I upload a model?
 
-You should consider uploading a model once you have trained and tested it in your own environment and you feel that it is production ready. Bailo is not intended to hold models until they have reached this point, so any development must be carried out in an external environment.
+You should consider uploading a model once you have trained and tested it in your own environment and you feel that it
+is production ready. Bailo is not intended to hold models until they have reached this point, so any development must be
+carried out in an external environment.
 
 ### What roles can a user have and how do they limit available actions?
 
-Currently, Bailo has **user** and **admin** roles. A **user** can upload and approve models (as both a manager and reviewer, as identifed during model upload). In the future, Bailo intends to support the following roles:
-
-- **Uploader**: This is the main role for users, it allows users to upload and explore models, as well as request deployments of models.
-- **Manager**: All models uploaded to Bailo are assigned a model manager. This person is responsible for the model and ensuring that it follows its lifecycle plan. The manager approves the upload of the model and all its deployments considering any business risks and costs associated with doing so.
-- **Reviewer**: Reviewer: All models uploaded to Bailo are assigned a model reviewer. The reviewer approves the upload of the model from a technical perspective considering the development details, the performance evaluation, assessed risks, limitations, bias and ethical considerations.
+Currently, Bailo has **user** and **admin** roles. A **user** can upload and approve models (as both a manager and
```

Comment:
The notion of "roles" needs to be discussed. I don't think our current version of Bailo has any concept of roles, and that anyone can be added as a reviewer/manager, and it is up to the uploader/requester to make sure they're adding the correct people.

In the future this might change of course but I think this part of the docs is a bit out of date.